shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
No definitive conclusions from user testing other than it's a somewhat unclear what "Identity providers" means. I would expect "Login providers" to be more clear to users.